.github/workflows/pypi-smoke-tests.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will install TSC from pypi and validate that it runs. For more information see:
+# https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Pypi smoke tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 11 * * * # Every day at 11AM UTC (7AM EST)
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.x']
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Set up Python ${{ matrix.python-version }} on ${{ matrix.os }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: pip install Tabcmd
+      run: |
+        pip uninstall tableauserverclient
+        pip install tableauserverclient
+    - name: Launch app
+      run: |
+        python -c "import tableauserverclient as TSC
+        server = TSC.Server('Fake-Server-Url', use_server_version=False)"
```

Comment:
For my understanding, how does this look when it fails?

Reply:
I'm not certain, because sometimes the github environment does something differently to what I expect. But it should check the exit code of the call to python: if any exceptions are thrown that will be something different to 0, and will be called a failed step.